src/split-pane/index.ts

```diff
@@ -55,6 +56,23 @@ export class SplitPaneBase<P extends SplitPaneProperties = SplitPaneProperties>
 	private _lastSize?: number;
 	private _position = 0;
 	private _collapsed = false;
+	private _resizeResultOverridden = false;
+	private _width = 0;
+
+	@diffProperty('collapseWidth')
+	@diffProperty('direction')
+	private collapseWidthDiff(oldProperties: SplitPaneProperties, { collapseWidth, direction, onCollapse }: SplitPaneProperties) {
+		if (collapseWidth) {
+			if (direction === Direction.row || collapseWidth < this._width && this._collapsed) {
+				this._collapsed = false;
+				this._resizeResultOverridden = true;
+			} else if (collapseWidth > this._width && !this._collapsed) {
```

Comment:
I'm new here so not sure about this but the style guide says the closing brace of `if` shouldn't be cuddled with the `else if` statement

Reply:
I think we may need to update those style guides. To be honest, most of our repo's use `prettier` now.